admin-api/index.php

```diff
@@ -70,6 +70,15 @@
 // When using the HttpCache, you need to call the method in your front controller instead of relying on the configuration parameter
 //Request::enableHttpMethodParameterOverride();
 $request = Request::createFromGlobals();
+
+/*
+ * Initialize legacy dispatcher at the initial stage of the request. If we don't do it now,
+ * the dispatcher could be created later by legacy classes. But, at that point, the request
+ * could already be modified, for examply by move_uploaded_file. That would cause createFromGlobals
+ * to crash.
+ */
+Dispatcher::getInstance($request);
```

Comment:
@Hlavtox I'm not sure if we need to do it here.

Reply:
I don't think there will be a situation where we use `move_uploaded_file` in the API context.

@jolelievre ?